ha/setup/ha_setup.py

```diff
@@ -244,7 +244,7 @@ def process(self):
         self._update_env(node_name, node_type, const.HA_CLUSTER_SOFTWARE)
         self._fetch_fids()
         self._update_cluster_manager_config()
-        self._cluster_manager = CortxClusterManager()
+        self._cluster_manager = CortxClusterManager(False)
```

Comment:
CortxClusterManager(logfix=False)
if there are more parameter updated in future then above can handle condition

Reply:
Done.